src/pages/SubmissionConfig.js

```diff
@@ -58,7 +58,7 @@ export const SubmissionConfig = ({ methodId }) => {
       const searchResult = await Ajax(signal).Wds.search.post(recordType)
       setRecords(searchResult.records)
     } catch (error) {
-      setNoRecordTypeData(`Data table not found: ${recordType}`)
+      setNoRecordTypeData('Select a data table')
```

Comment:
So I think we want to keep the `Data table not found: ${recordType}` if there is a `recordType` to show. But if it's null or undefined, we probably want to show the `'Select a data table'` version.